sdk/trace/span.go

```diff
@@ -417,9 +429,10 @@ func (s *span) Snapshot() *export.SpanSnapshot {
 	sd.StatusCode = s.statusCode
 	sd.StatusMessage = s.statusMessage
 
+	sd.DroppedAttributeCount = int(s.droppedAttributeCount)
```

Comment:
We should add some documentation to [this field](https://github.com/open-telemetry/opentelemetry-go/blob/3bce9c97f800c12a791e78a1611991818f9b4db6/sdk/export/trace/trace.go#L71) to make sure users know it will contain not only dropped attributes for the span itself, but also for any attributes for events and links as well.